_unittest/test_45_FilterSolutions/test_lumped_export/test_export_to_aedt.py

```diff
@@ -450,7 +450,7 @@ def test_load_library_parts_config(self):
         lumpdesign = ansys.aedt.core.FilterSolutions(implementation_type=FilterImplementation.LUMPED)
         lumpdesign.export_to_aedt._open_aedt_export()
         lumpdesign.export_to_aedt.load_library_parts_config(resource_path("library_parts.cfg"))
-        assert lumpdesign.export_to_aedt.part_libraries == PartLibraries.MODELITHICS
+        lumpdesign.export_to_aedt.part_libraries == PartLibraries.MODELITHICS
```

Comment:
@ramin4667 I'm not sure what's intended by this line. It does an equivalence check still but does not include the assert.